docs/antora-playbook-dev.yml

```diff
@@ -26,7 +26,7 @@ content:
     start_path: docs
   - url: git@github.com:apache/camel.git
     branches:
-      - camel-3.13.x # replace ${camel.docs.branch}
+      - camel-3.14.x # replace ${camel.docs.branch}
```

Comment:
I thought `antora-playbook-dev.yml` and `antora-playbook` were not needed anymore with this [recent change](https://github.com/apache/camel-quarkus/commit/43cede1eca3fc257387ba448b371228df9669044).

Reply:
There was a problem with rebase. Thanks for catching this